lib/es-models/bib.js

```diff
@@ -256,7 +258,11 @@ class EsBib extends EsBase {
         tag: type
       })
     }
-    return dates.filter(x => x)
+    const filteredDates = dates.filter(x => x)
+    if (!filteredDates.length) {
+      logger.warn(`Bib missing dates: ${id}`)
+    }
```

Comment:
I think I agree with your concern that this is going to generate a lot of noise.